dbt_semantic_interfaces/protocols/semantic_model.py

```diff
@@ -155,5 +156,14 @@ def checked_agg_time_dimension_for_measure(self, measure_reference: MeasureRefer
         """
         ...
 
+    @property
```

Comment:
Looks like we're missing `primary_entity: Optional[str]` on the protocol

Reply:
If you look at this PR in isolation, no. However so far our approach has been that protocols have a number of attributes and some of those things should have a reference counter part that is exposed. We've actually discussed a few times whether the `*_reference` properties or if there should be a generic utility which can take in a DSI protocol object and expose the relevant `references`. Perhaps we decide later on to prioritize references in the protocol over the attributes that result in them, that references shouldn't be required properties of the protocol spec, or that both should continue to exist. But we need to make that decision, and we shouldn't being making a paradigm shift like that during the RC for a single attribute. DSI 0.1.0 should at least be consistent and thus for 0.1.0 exposes both.